service/src/main/java/uk/gov/hmcts/reform/fpl/service/UserService.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityLookupConfiguration;
+import uk.gov.hmcts.reform.idam.client.IdamApi;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+import java.util.Map;
+
+/**
+ * Gets Users from Idam.
+ */
+@Service
+public class UserService {
+
+    private final IdamApi idamApi;
+    private final LocalAuthorityLookupConfiguration localAuthorityLookupConfiguration;
```

Comment:
Shall we rename it to be `LocalAuthorityNameLookupConfiguration`? Again current name is a bit generic.